packages/dd-trace/src/proxy.js

```diff
@@ -24,6 +39,12 @@ class Tracer extends NoopProxy {
     this._pluginManager = new PluginManager(this)
     this.dogstatsd = new NoopDogStatsDClient()
     this._tracingInitialized = false
+
+    // this requires must work with esm bundler
```

Comment:
plural for `this` is `these`
```suggestion
    // these requires must work with esm bundler
```